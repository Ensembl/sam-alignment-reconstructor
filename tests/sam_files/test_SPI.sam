r002	0	ref	9	30	1S2I6M1P1I1P1I4M2I	*	0	0	AAAAGATAAGGGATAAA	*	NM:i:6	MD:Z:10
