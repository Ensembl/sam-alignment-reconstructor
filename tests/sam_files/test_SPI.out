r002
aAAAGATAA*G*GGATAAA
   ||||||    ||||  
.--AGATAA*-*-GATA--

